configs/taco/NJ.yaml

```diff
@@ -2,16 +2,13 @@ state: NJ
 
 links: 
 - name: primary
-  url: https://covid19.nj.gov/#live-updates
+  url: https://njhealth.maps.arcgis.com/apps/MapSeries/index.html?appid=50c2c6af93364b4da9c0bf6327c04b45&folderid=e5d6362c0f1f4f9684dc650f00741b24
   overseerScript: |
     page.manualWait();
-    await page.waitForDelay(30000);
-    await page.waitForSelector("button#overlay-close");
-    page.click("button#overlay-close");
-    await page.waitForDelay(30000);
+    await page.waitForDelay(60000);
```

Comment:
The script logic times out after 60 sec by default so we could add the following to dictate it to wait 60 seconds:

```
  requestSettings: 
    maxWait: 60000
```

Reply:
yes! I always forget about the max wait on the 60 second ones

Reply:
Same haha. I think that your IA PR has the same comment too